tests/helperDb.js

```diff
@@ -114,46 +115,58 @@ async function databaseExists(dbName) {
 }
 
 async function setupDatabases(mochaThis) {
-  const exists = await databaseExists(postgresqlDatabaseTemplate);
-  if (exists) {
-    await createFromTemplate(postgresqlDatabase, postgresqlDatabaseTemplate, true, mochaThis);
+  const templateExists = await databaseExists(POSTGRES_DATABASE_TEMPLATE);
+  const dbName = module.exports.getDatabaseNameForCurrentWorker();
+  if (templateExists) {
+    await createFromTemplate(dbName, POSTGRES_DATABASE_TEMPLATE, true, mochaThis);
   } else {
-    await createFullDatabase(postgresqlDatabaseTemplate, true, mochaThis);
-    await createFromTemplate(postgresqlDatabase, postgresqlDatabaseTemplate, true, mochaThis);
+    await module.exports.createTemplate(mochaThis);
+    await createFromTemplate(dbName, POSTGRES_DATABASE_TEMPLATE, true, mochaThis);
   }
-  await establishSql(postgresqlDatabase);
+  await establishSql(dbName);
 }
 
-module.exports = {
```

Comment:
Why this change? a style guide thing or something?

Reply:
personal preference slash better symmetry with ESM `import`/`export` syntax that we'll hopefully move to soonish!